job-dsls/jobs/deploy_jobs.groovy

```diff
@@ -138,8 +138,12 @@ def final REPO_CONFIGS = [
         "jbpm-work-items"           : [
                 label      : "linux && mem4g",
                 timeoutMins: 30,
-                ircNotificationChannels: ["#jbpmdev"]
-        ]
+                ircNotificationChannels: ["#jbpmdev"],
+                downstreamRepos        : ["optashift-employee-rostering"]
```

Comment:
optashift-employee-rostering is a downstream repo of optaplanner, not of jbpm-work-items?

Reply:
Ah, but I get it, it is also a downstream repo of errai, so this is the first repo where optaplanner and errai are downstream of.